### PR TITLE
PB-103 : adding two new utils to help create getFeatureInfo requests

### PR DESCRIPTION
**Pull request by pakb**
An extent utils to create some arbitrary sized extent around a given coordinate A GeoJSON utils that extract a single coordinate from any type of feature (to position the tooltip correctly)

[Test link](https://sys-map.dev.bgdi.ch/preview/feat_pb-103_add_two_utils/index.html)

### CODE REVIEW

File: src/utils/extentUtils.js [hunk 1]
```diff
@@ -33,3 +38,45 @@ export function getExtentForProjection(projection, extent) {
...
+/**
+ * @param {Number} config.size Number of pixels the extent should be (if 100 is given, a box of
+ *   100x100 pixels with the coordinate at its center will be returned)
+ * @param {[Number, Number]} config.coordinate Where the center of the 100pixel extent should be.
+ * @param {CoordinateSystem} config.projection Projection used to describe the coordinates
+ * @param {Number} config.resolution Current map resolution, necessary to calculate how much
+ *   distance 100pixel means.
+ * @param {Boolean} [config.rounded=false] Tells if the extent's value should be rounded before
+ *   being returned. Default is `false`
+ * @returns {[[Number, Number], [Number, Number]]} A squared extent of 100pixel around the given
```
Comment (ltshb) [inline src/utils/extentUtils.js:51]: or `null` in case invalid parameter
`A squared extent of "size" pixel(s)`

File: src/utils/extentUtils.js [hunk 2]
```diff
@@ -33,3 +38,45 @@ export function getExtentForProjection(projection, extent) {
     }
     return null
 }
+
+/**
+ * @param {Number} config.size Number of pixels the extent should be (if 100 is given, a box of
+ *   100x100 pixels with the coordinate at its center will be returned)
+ * @param {[Number, Number]} config.coordinate Where the center of the 100pixel extent should be.
```
Comment (ltshb) [inline src/utils/extentUtils.js:45]: `of the pixel extent`

File: src/utils/extentUtils.js [hunk 3]
```diff
@@ -33,3 +38,45 @@ export function getExtentForProjection(projection, extent) {
...
+    let coordinatesWgs84 = coordinate
+    if (projection.epsg !== WGS84.epsg) {
+        coordinatesWgs84 = proj4(projection.epsg, WGS84.epsg, coordinate)
+    }
+    let extent = bbox(
+        buffer(
+            point(coordinatesWgs84),
+            // sphere of the wanted number of pixels as radius around the coordinate
+            size * resolution,
+            { units: 'meters' }
```
Comment (ltshb) [inline src/utils/extentUtils.js:68]: I don't know the turf library but it seems strange to use `meter` with WGS84 ?

**Review by ltshb — APPROVED** [review]
Good just some minor comments

File: src/utils/extentUtils.js (hunk 3 above)
Comment (pakb) [reply src/utils/extentUtils.js:69]: this lib can do quite a lot of unit conversion

**Pull request merged**